### ISSUE TITLE
fatal error using ansible playbook

### ISSUE THREAD

**Issue by jhb2** [andrw3000/vespai#5]
Hello,

I'm using the ansible playbook to deploy vespAI, on the control node when I launch : 

ansible-playbook /home/jhb/vespai/vespai-git/deploy/rpi-playbook.yml --inventory=192.168.1.60, --extra-vars "user=jhb vespai_repo=https://github.com/andrw3000/vespai.git vespai_branch=ansible-deployment"

I got:

PLAY [all] *******************************************************************************************************************************************************************************

TASK [Gathering Facts] *******************************************************************************************************************************************************************
[WARNING]: Platform linux on host 192.168.1.60 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter could change the
meaning of that path. See https://docs.ansible.com/ansible-core/2.17/reference_appendices/interpreter_discovery.html for more information.
ok: [192.168.1.60]

TASK [Update apt repo cache and install git.] ********************************************************************************************************************************************
ok: [192.168.1.60]

TASK [Download VespAI git repo; recurse/fetch and track/update submodules.] **************************************************************************************************************
fatal: [192.168.1.60]: FAILED! => {"changed": false, "cmd": "/usr/bin/git checkout --force ansible-deployment", "msg": "Failed to checkout ansible-deployment", "rc": 1, "stderr": "error: pathspec 'ansible-deployment' did not match any file(s) known to git\n", "stderr_lines": ["error: pathspec 'ansible-deployment' did not match any file(s) known to git"], "stdout": "", "stdout_lines": []}

PLAY RECAP *******************************************************************************************************************************************************************************
192.168.1.60               : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

The fatal error is of course the issue, what can I do to suppress it?

Thank you.

**Comment by wombat70** [comment]
Hi - apologies - this was probably due to a slighty misleading example I gave in the [deployment README](https://github.com/andrw3000/vespai/blob/57eb2ee8f8a157603fd7e90135735294985c883a/deploy/README.md?plain=1#L36). _The branch name that was used in the example was from my forked repo, where I was making the edits for pull-request #3._

Now that the PR is merged to this repo, the example could be rewritten with:
    - `<git-repo-branch>` is the branch-name (or other git ref) in the git repo, e.g. `main`

In other words, you could try running:
```
ansible-playbook deploy/rpi-playbook.yml --inventory=<rpi-host>, --extra-vars "user=<username> vespai_repo=https://github.com/andrw3000/vespai.git vespai_branch=main"
```

Just created new PR #7 to update README.
